src/components/Workspace.tsx

```diff
@@ -47,7 +47,9 @@ export default function Workspace() {
   const [swcVersion] = useAtom(swcVersionAtom)
   const { data: swc, error } = useSWR(swcVersion, loadSwc)
   const [code] = useAtom(codeAtom)
+  const deferredCode = useDeferredValue(code)
```

Comment:
Don't make the code complex and pedantic by adding unnecessary code which just looks "faster".

Reply:
> Don't make the code complex and pedantic by adding unnecessary code which just looks "faster".

Oops sorry && ok 💪🙌